util/gh-pages/style.css

```diff
@@ -272,8 +272,6 @@ L4.75,12h2.5l0.5393066-2.1572876  c0.2276001-0.1062012,0.4459839-0.2269287,0.649
   height: 18px;
   display: block;
   filter: invert(0.7);
-  padding-left: 4px;
-  padding-top: 3px;
```

Comment:
Strangely enough, if I put it back I get:

![image](https://github.com/user-attachments/assets/05cdb39c-8ffd-4975-a735-b8f6853258a9)

Reply:
It is more noticeable in chrome (the UA stylesheet for button padding is different)
![image](https://github.com/user-attachments/assets/c242a83f-1c78-40bb-876a-dc68cd74bcef)

But the image is slightly below centre in firefox also, before/after: 
![image](https://github.com/user-attachments/assets/b55b935d-1429-4035-96e1-ad022ccc0bb7)
![image](https://github.com/user-attachments/assets/c0084095-64e9-4f21-a554-f688c2de60ea)

Reply:
Hum. Let me check if I can unify this across browsers.

Reply:
Instead of having something set by the default, I now force its position by using absolute positioning. I checked on firefox and chrome and both look the same now.